meta-ostro/classes/ostro-image.bbclass

```diff
@@ -381,6 +382,39 @@ inherit xattr-images
 # Create all users and groups normally created only at runtime already at build time.
 inherit systemd-sysusers
 
+# Provide an image feature that disables all consoles and makes
+# journald mute.
+python() {
+    if bb.utils.contains('IMAGE_FEATURES', 'muted', True, False, d):
+        import re
+
+	# Mangle cmdline to drop all consoles and add quiet option
+        cmdline = d.getVar('APPEND', True)
+        new=re.sub('console=\w+(,\w*)*\s','', cmdline) + ' quiet'
+        d.setVar('APPEND', new)
+}
+
+# In addition to cmdline mangling, make sure journal only gets
+# emergency errors and any lower priority messages do not get
+# logged. This is temporary approach and the cleanest way to disable
+# logging (compared with removing the journald binary and related
+# services here). Once the systemd recipe gives us better granulatiry
+# for packaging, the preferred way to avoid logging is
+# to not install systemd-journald at all.
+muted () {
```

Comment:
I'd use "ostro_image_muted" here. Makes it easier to grep for it and avoids namespace issues (should someone else decide to add something called "muted").

Some of our other definitions in ostro-image.bbclass set a bad precedent here by using longer names but without consistent prefix.